src/core/dbus-cgroup.c

```diff
@@ -396,6 +398,7 @@ const sd_bus_vtable bus_cgroup_vtable[] = {
         SD_BUS_PROPERTY("ManagedOOMSwap", "s", property_get_managed_oom_mode, offsetof(CGroupContext, moom_swap), 0),
         SD_BUS_PROPERTY("ManagedOOMMemoryPressure", "s", property_get_managed_oom_mode, offsetof(CGroupContext, moom_mem_pressure), 0),
         SD_BUS_PROPERTY("ManagedOOMMemoryPressureLimitPercent", "s", bus_property_get_percent, offsetof(CGroupContext, moom_mem_pressure_limit), 0),
+        SD_BUS_PROPERTY("AllowBindPorts", "as", NULL, offsetof(CGroupContext, allow_bind_ports), 0),
```

Comment:
I'm getting 
```
$ sudo systemctl status myservice
Failed to get properties: Bad message
```

when trying to get the status of an unit defining `AllowBindPorts=5000`. 

If `allow_bind_ports` is a `Set`, shouldn't it define a custom property getter method? 
(Please ignore if my comment doesn't make any sense, I'm new to dbus :) )